.github/workflows/build.yml

```diff
@@ -130,3 +134,8 @@ jobs:
         path: "*.log"
     - name: Report check status
       run: exit $(< make-check.status)
+    - name: Check expected plugins
+      run: |
+        if [[ -e ".expected-plugins/${{ matrix.container_tag }}" ]]; then
+          /checks/check-built-plugins.sh ".expected-plugins/${{ matrix.container_tag }}"
+        fi
```

Comment:
For some containers, no plugin checks are done?

Reply:
See above.